proto/zitadel/user/v2beta/user_service.proto

```diff
@@ -931,6 +931,14 @@ message AddHumanUserRequest{
     HashedPassword hashed_password = 8;
   }
   repeated IDPLink idp_links = 9;
+  optional string totp_secret = 12 [
+    (validate.rules).string = {min_len: 1, max_len: 200},
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      min_length: 1;
+      max_length: 200;
+      example: "\"TJOPWSDYILLHXFV4MLKNNJOWFG7VSDCK\"";
+    }
+  ];
```

Comment:
We should a some documentation / description here. For example we need to explain zitadel uses an implementation of [RFC 6238](https://datatracker.ietf.org/doc/html/rfc6238) with the HMAC-SHA-1 function and recommended time-step of 30 seconds.

This is important, as the source of the import may have used other options (other SHA or time-step) from the RFC and future validations will most certainly fail.

Reply:
Added a comment with RFC 6238, HMAC-SHA-1 and time-step of 30 seconds, with the hint that with other options the validation will fail.